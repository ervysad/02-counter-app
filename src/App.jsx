
import './App.css';
import PropTypes from 'prop-types';


function App(props) {
  return (
    <div className="App">
      <header>
        <h1>{ props.proping }</h1>
      </header>
    </div>
  );
}


export default App;




App.propTypes = {
  proping: PropTypes.string.isRequired,
}