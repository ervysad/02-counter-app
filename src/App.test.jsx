import { render } from "@testing-library/react";
import App from "./App";


describe('Pruebas en el componente titulo App.jsx', () => {

  test('Debe ser igual al snapshot definido', () => {

    const { container } = render( < App proping="testing"/> )
    expect( container ).toMatchSnapshot();
});
});