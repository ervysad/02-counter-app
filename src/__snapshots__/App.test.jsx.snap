// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente titulo App.jsx Debe ser igual al snapshot definido 1`] = `
<div>
  <div
    class="App"
  >
    <header>
      <h1>
        testing
      </h1>
    </header>
  </div>
</div>
`;
